src/game/logic/ai/aipathfind.cpp

```diff
@@ -16,6 +16,220 @@
 #ifdef GAME_DLL
 #include "hooker.h"
 #endif
+#include "xfer.h"
+
+PathfindZoneManager::PathfindZoneManager() :
+    m_zoneBlocks(nullptr),
+    m_zoneBlockPointers(nullptr),
+    m_maxZone(0),
+    m_updateFrequency(0),
+    m_zoneTableSize(0),
+    m_zoneTable1(nullptr),
+    m_zoneTable2(nullptr),
+    m_zoneTable3(nullptr),
+    m_zoneTable4(nullptr),
+    m_zoneTable5(nullptr),
+    m_zoneTable6(nullptr)
+{
+    m_blockSize.x = 0;
+    m_blockSize.y = 0;
+}
+
+PathfindZoneManager::~PathfindZoneManager()
+{
+#ifdef GAME_DLL
+    Call_Method<void, PathfindZoneManager>(PICK_ADDRESS(0x0055C320, 0x0088BF4E), this);
+#endif
+}
+
+PathfindLayer::PathfindLayer() : m_map(nullptr), m_mapPointers(nullptr), m_unk(0), m_bridge(nullptr), m_destroyed(false)

```

Comment:
`PathfindLayerEnum m_layerNum` not initialized.

Reply:
fixed.